config/config.go

```diff
@@ -487,59 +513,6 @@ func decodeAsTerragruntConfigFile(
 	return &terragruntConfig, nil
 }
 
```

Comment:
These functions were refactored out into hcl_parser.go so all the HCL parsing related functions were consolidated.